README.md

```diff
@@ -213,7 +213,7 @@ Learn about full lifecycle of cells in Cellery from,
 * [Developing and runing a Cell](docs/writing-a-cell.md) - step by step explanation on how you could define your own cells.
 * [Testing a Cell](docs/cell-testing.md) - provides the instructions to run test against cells.
 * [Managing a Cell](docs/cell-management.md) - lists the management aspects of cells in the runtime.
-    * [Update](docs/cell-update.md) - update the running cell instance with the new version.
+    * [Update](docs/cell-update-and-adv-deployment.md) - update the running cell instance with the new version.
```

Comment:
As you have renamed the small description in other sections, can you please update this as well?